src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -60,11 +62,11 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef(
       maxContentWidth,
       placement,
       ...rest
-    }: AppLayoutPropsWithDefaults,
-    forwardRef: React.Ref<AppLayoutProps.Ref>
+    },
+    forwardRef
   ) => {
     const isMobile = useMobile();
-    const embeddedViewMode = (rest as any).__embeddedViewMode;
+    const { __embeddedViewMode: embeddedViewMode, __forceDeduplicationType: forceDeduplicationType } = rest as any;
```

Comment:
New private property here, `__forceDeduplicationType`, allows to override automatic deduplication behavior